addons/godot_tours/core/overlays/overlays.gd

```diff
@@ -36,22 +37,25 @@ var _highlight_style_scaled: StyleBoxFlat = null
 
 func _init(interface: EditorInterfaceAccess, editor_scale := 1.0) -> void:
 	self.interface = interface
-
-	for control: Control in [interface.base_control, interface.signals_dialog, interface.node_create_panel]:
+	for viewport: Viewport in interface.viewports:
```

Comment:
As mentioned in interface access, I'd see if we can't just create dimmers as needed, when reaching a step that highlights a node that targets a new viewport in a tour.

In Godot 4.2 you should be able to get the editor base viewport with the singleton EditorInterface: `EditorInterface.get_base_control().get_viewport()` off the top of my head